test/TEST-19-DELEGATE/test.sh

```diff
@@ -5,6 +5,9 @@ set -e
 TEST_DESCRIPTION="test cgroup delegation in the unifier hierarchy"
 TEST_NO_NSPAWN=1
 
+# Skip this test if the kernel doesn't actually support the cgroupsv2
+grep -q cgroup2 /proc/filesystems || exit 0
```

Comment:
This probably will work in most cases, but I'm not sure about a scenario where `KERNEL_BIN` is used to specify the kernel being used to run the test, because it can be completely different from the host kernel.

Reply:
I think it would be better to place this check at the bottom of `testsuite.sh`.

Reply:
Yeah. Also, please also print a message before exiting.

Reply:
i wanted to avoid that we even run the test when cgroupv2 is not around...

Reply:
Yeah, but we need to check the qemu kernel, not the host one. Also the host could have some boot option to disable cgroups2, but the guest not.